docs/OPERATIONS.md

```diff
@@ -0,0 +1,274 @@
+    (treasury.ychad.eth)
+
+- Check new vault has ABI setup on etherscan (until verification with Vyper and proxy is fixed on Etherscan).
+- Coordinate with core developer to set proper deposit limit and other settings for new vault. See the table below: [Limits per Stage](#limits-per-stage).
+- Set a deposit limit to \$50k USD converted to your `want` token. Example below is 50k DAI
+
+  ```python
+  vault.setDepositLimit(50_000 * 1e18)
+  ```
+
+- Deploy strategy and upload code to Etherscan for verification.
+- Tag GitHub review issue with deployed version and add mainnet address(es).
+
+## After deployment
+
+- Add strategy to vault:
+
+  ```python
+  vault.addStrategy(strategy, debt_ratio, rate_limit, 1000)
+  ```
+
+  - `debt_ratio` should be `9800` if first strategy on vault.
+  - `rate_limit` is `0` unless there is reason for it to be different.
+
+- Set keeper:
+
+  ```python
+  strategy.setKeeper(keep3r_manager)
+  ```
+
+  - `keep3r_manager` = `0x13dAda6157Fee283723c0254F43FF1FdADe4EEd6`
+
+- Set rewards:
+
+  ```python
+  strategy.setRewards(address)
+  ```
+
+  - Read [below](<#Sharer-contract-(optional)>) if you want to use the sharer contract.
+
+- Set management fee to 0:
+
+  ```python
+  vault.setManagementFee(0)
+  ```
+
+- Set governance
+
+  ```python
+  vault.setGovernance(multisig)
+  ```
+
+  - `multisig` = `0x846e211e8ba920b353fb717631c015cf04061cc9`
+    (`dev.ychad.eth`).
+  - Governance needs to be accepted before it is in place. After you set this you will still have control over the strategy.
+
+- Run tests against "live" vault and strategy in mainnet-fork:
+
+  - Harvest.
+  - Profitable harvest.
+  - Revoke strategy and check that funds return to the vault.
+  - Increase/decrease debt + harvest, and check that the strategy is working well.
+  - Migration.
+  - Check that tokens in the strategy cannot be sweeped by dust collection.
+
+  Example: Hegic strat [repo](https://github.com/Macarse/yhegic/tree/master/tests/development).
+
+- Tag vault as "experimental" in v2.registry.ychad.eth
+  - `registry.tagVault(vaultAddr, "https://meta.yearn.network/vaults/${vaultAddr}/vault.json")`
+
+### Sharer contract (optional)
+
+"Sharer" is a contract for distributing/splitting strategist rewards. For boarding school graduates suggested split is 34% to strategist multisig and 66% to strategist – [repo](https://github.com/Grandthrax/Sharer).
+
+- Setup rewards for your strategy by calling `sharer.addContributors`.
+- Include devs if you forked someone else's strategy.
+- Be sure to reward people who helped you.
+
+### Example Script
+
+```python
+# TBD Fill CMO example script to fill
+```
+
+## The Manual Phase
+
+- [ ] Deposit some `want` tokens into the vault.
+- [ ] Do first `harvest` and make sure it worked correctly.
+
+  ```python
+  strategy.harvest()
+  ```
+
+- [ ] Publish on yearn.rocks:
+  - Governance dev multisig **_must_** call `vault.acceptGovernance()` first.
+  - Talk to Facu.
```

Comment:
How do I contact him?

Reply:
Removed. Added instructions to create a PR.